lib/pages/settings/view/settings_page.dart

```diff
@@ -96,6 +111,36 @@ class SettingsPageState extends State<SettingsPage> {
                                 context: context),
                           ),
                         ),
+                        const Divider(),
+                        FlatButton(
+                          key: const ValueKey('ask_permissions'),
+                          onPressed: () => {
+                            if (!authProvider.isVerifiedFromCache &&
+                                !authProvider.isAnonymous)
+                              {
+                                AppToast.show(S
+                                    .of(context)
+                                    .messageEmailNotVerifiedAskPermissions)
+                              }
+                            else if (!authProvider.isAnonymous)
+                              {
+                                Navigator.of(context).push(
+                                    MaterialPageRoute<AskPermissions>(
+                                        builder: (context) =>
```

Comment:
Nit: You don't need the context here
```suggestion
                                        builder: (_) =>
```